patches/chrome-android-java-src-org-chromium-chrome-browser-ChromeTabbedActivity.java.patch

```diff
@@ -20,6 +20,15 @@ index 1e8697a4a42e061cf4397f40fb62b698ce75f567..a5d1f5044c0bd31cbba83881ba33ec81
                              assert profile != null : "Unexpectedly null profile from TabModel.";
                              if (profile == null) return;
  
+@@ -1425,7 +1425,7 @@ public class ChromeTabbedActivity extends ChromeActivity<ChromeActivityComponent
+ 
+     @Override
+     protected RootUiCoordinator createRootUiCoordinator() {
+-        return new TabbedRootUiCoordinator(this, this::onOmniboxFocusChanged,
```

Comment:
all of these patches seem like they can be done with asm

Reply:
It requires additional fix in bytecode, issue created https://github.com/brave/brave-browser/issues/12284